NDcPP_v3_0.adoc

```diff
@@ -2280,7 +2280,7 @@ _The following table provides the ST author with instructions for completing the
 
 _The option 'provides' shall be selected if the TOE provides the ability to configure the list of ciphers as defined in FCS_DTLSC_EXT.1.1 (e.g. enabling/disabling of ciphers, ordering, assigning priorities). Otherwise, the option 'does not provide' shall be selected. If the 'provides' option is selected by the ST Author, the ST Author also selects the corresponding function in FMT_SMF.1._
 
-*FCS_DTLSC_EXT.1.7* The TSF _shall prohibit the use of the following extensions:
+*FCS_DTLSC_EXT.1.7* The TSF shall prohibit the use of the following extensions:
```

Comment:
@Kirky-J FCS_DTLSS_EXT.1.9, FCS_TLSC_EXT.1.7, FCS_TLSS_EXT.1.6 has the same wording italicized.

Reply:
Removed incorrect formatting on these SFRs. Thanks!

Reply:
@Kirky-J There's another instance of FCS_DTLSS_EXT.1.9, FCS_TLSC_EXT.1.7, FCS_TLSS_EXT.1.6 that still needs the italics fixed. Let me know if you want me to make the change.

Reply:
Ok. Got them now too!